### ISSUE TITLE
"aria_" should be handled like "data_" in html_params

### ISSUE THREAD

**Issue by MM1nd** [wtforms/wtforms#239]
The title says it. Naturally this can be done by changing  

```
elif k.startswith('data_') 
```

to 

```
elif k.startswith('data_') or k.startswith('aria_')
```

but in the long run probably a more sophisticated system might be needed. Maybe a sophisticated implementation already exists.

Related Issue (https://github.com/wtforms/wtforms/issues/48)

Further reference if needed: http://www.w3.org/TR/wai-aria/

Alex

**Comment by hammygoonan** [comment]
+1

Further reference here as well: [https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA)

This is a feature I'd really like to see implemented.

It's now part of Foundation's standard help-text formatting as well [http://foundation.zurb.com/sites/docs/forms.html#help-text](http://foundation.zurb.com/sites/docs/forms.html#help-text)

**Comment by crast** [comment]
Good idea. I'm putting it on the schedule for 3.0

**Comment by ftm-1** [comment]
@crast Is this feature anywhere near to being included? I have a PR ready to submit if this is still something you guys want?

**Comment by davidism** [comment]
@ftm-1 I'm the new maintainer. If you have a PR, please submit it! :-)

**Comment by ftm** [comment]
Hi @davidism, I was @ftm-1 but now I'm on this account (I managed to get a better username). I have reopened the PR on my new account so it should be ready to go.

**Comment by davidism** [comment]
Closed in #389
